pkg/chevronlib/base.go

```diff
@@ -4,16 +4,20 @@ import (
 	"context"
 	"github.com/quan-to/chevron/internal/etc/magicbuilder"
 	"github.com/quan-to/chevron/internal/tools"
+	"github.com/quan-to/chevron/pkg/database/memory"
 	"github.com/quan-to/chevron/pkg/interfaces"
 	"github.com/quan-to/slog"
```

Comment:
Same comment for [imports](https://github.com/golang/go/wiki/CodeReviewComments#imports).